includes/blocks/class-sensei-block-quiz-progress.php

```diff
@@ -43,7 +43,7 @@ public function render( $attributes ): string {
 
 		$quiz_id = get_the_ID();
 
-		$total         = count( Sensei()->quiz->get_questions( $quiz_id ) );
+		$total         = (int) get_post_meta( $quiz_id, '_show_questions', true );
```

Comment:
Would this work if the 'Number of Questions' is not set? I think it'll need to fall back to the get_questions count in that case.
Also it could be wrong when 'Number of Questions' is set to higher than the number of questions in the quiz.

Reply:
Good catch, @yscik - I've updated now so we have a fallback in case the number of question's is not set.   I tried making the `Number of Questions` higher than the questions in the quiz and it looks like there's good validation on the editor that won't let me do it.
I even tried setting it to 20, saving, then removing a question, but the number of quesitons automatically went down to 19.